dags/search_forecasting.py

```diff
@@ -0,0 +1,58 @@
+"""
+See [kpi-forecasting in the docker-etl repository](https://github.com/mozilla/docker-etl/blob/main/jobs/kpi-forecasting).
+
+This DAG runs the search forecasts for the DAU, search count and ad clicks metrics .
+
+This DAG is high priority for week 1 of the month and low priority otherwise.
+"""
+
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+
+from operators.gcp_container_operator import GKEPodOperator
+from utils.tags import Tag
+
+default_args = {
+    "owner": "jsnyder@mozilla.com",
+    "email": ["jsnyder@mozilla.com", "mbowerman@mozilla.com"],
+    "depends_on_past": False,
+    "start_date": datetime(2024, 6, 21),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+TAGS = [Tag.ImpactTier.tier_1]
+IMAGE = "gcr.io/moz-fx-data-airflow-prod-88e0/kpi-forecasting_docker_etl:latest"
+
+FORECAST_METRICS_LIST = [
+    "search_forecasting_daily_active_users",
+    "search_forecasting_search_count",
+    "search_forecasting_ad_clicks",
+]
+
+with DAG(
+    "search_forecasting",
+    default_args=default_args,
+    schedule_interval="7 7 7 * *",
```

Comment:
Is this correct? Run once per month on the 7th at 7:07?

Reply:
Yeah :D

Reply:
7:07 might be too specific, but it should be running once a month on the 7th, yeah.

Reply:
my reviewers have no love of beauty, no appreciation of the silly, smgdh